csrankings-h.csv

```diff
@@ -668,6 +668,7 @@ Holger Hermanns,Saarland University,http://depend.cs.uni-saarland.de/index.php?h
 Holger Hoos,University of British Columbia,http://www.cs.ubc.ca/~hoos,l6c85tMAAAAJ
 Holger Karl,Paderborn University,https://cs.uni-paderborn.de/cn/team/staff/karl,vjgKHWMAAAAJ
 Holger Pirk,Imperial College London,https://www.doc.ic.ac.uk/~hlgr,1jP_gqcAAAAJ
+Holger Regenbrecht,University of Otago,https://www.otago.ac.nz/info-science/people/staff/holger-regenbrecht.html,RyTlTtoAAAAJ

```

Comment:
Info Sciences, not CS, not listed here: https://www.otago.ac.nz/computer-science/people/index.html. Please note inclusion eligibility requires the ability to solely advise PhD students for a Phd in _Computer Science_ or the equivalent - please remove if ineligible or explain how they meet the criteria. Thanks.

Reply:
There are two computing departments here in Otago that work very closely together, sharing teaching and PhD supervision. These are Computer Science and Information Science. Holger Regenbrecht is an internationally well-known expert in VR and HCI, both categroies in the csranking. So I think he needs to be included otherwise the csranking would not reflect the expertise in Otago.

Reply:
Per the instructions, emphasis mine:

>Eligible faculty include only full-time, tenure-track research faculty members on a given campus who can solely advise PhD students in Computer Science. Faculty not in a CS department or similar who can advise PhD students **in CS** can be included regardless of their home department.

Eligibility for faculty inclusion is entirely based on whether they can solely supervise a PhD in the Computer Science department.